src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
"copy" does have the advantage of imitating the JES backend more precisely since there is no relation to an existing file anywhere else (it's possible tools might try to resolve to a "canonical" file and get weird results).  But if you do decide to keep this as is, perhaps there should be some comments to capture the thinking behind this choice.

Also not clear why this is soft -> hard, while inputs are hard -> soft